app/src/main/java/com/randomappsinc/studentpicker/presentation/PresentationActivity.java

```diff
@@ -85,19 +79,17 @@ protected void onCreate(Bundle savedInstanceState) {
         settings = dataSource.getChoosingSettings(listId);
         textToSpeechManager = new TextToSpeechManager(this, this);
 
+        presentationManager = new PresentationManager(this, namesList);
+        DividerItemDecoration itemDecorator =
+                new DividerItemDecoration(this, DividerItemDecoration.VERTICAL);
```

Comment:
I am unsure how to feel about the divider. I know it's in the dialog, but on this page, it just seems... off.